libs/wire-api/src/Wire/API/Error/Cargohold.hs

```diff
@@ -38,5 +40,9 @@ type instance MapError 'AssetTooLarge = 'StaticError 413 "client-error" "Asset t
 
 type instance MapError 'InvalidLength = 'StaticError 400 "invalid-length" "Invalid content length"
 
+type instance MapError 'Unverified = 'StaticError 403 "unverified" "Unverified"
+
+type instance MapError 'UserNotFound = 'StaticError 404 "not-found" "User not found"
+
```

Comment:
Semantically this should be a 403 since the HTTP operation is not on the user resource but on the asset resource.

Reply:
Good catch. I'll update it and any tests that depend on it.